### DISCUSSION TITLE
Kindly Updated  Phalcon 5.0 Documentation for UBUNTU 20.04 , PHP 8.0 , Mysql 8x.

### DISCUSSION THREAD

**Discussion by RajuHarry** [phalcon/cphalcon#15743] [Q&A] [answered]
Unable to find Any installation For Ubuntu |  Apache / Nginx 
Want Step by step Guide to install Phalcon on  UBUNTU 20.04 , PHP 8.0

Please do it  Awaiting for Phalcon 5 from last 6 Months.
Phalcon Should continue in Future . 
This one is the best Framework in PHP . Using it From last 7 Years .
Built many Powerful Websites because of it .

**Comment by RajuHarry** [comment, marked as answer]
@Jeckerson  With the Help Of You and My Research I could Found Below Method For Phalcon Community.


sudo apt-get update
sudo apt-get install apache2
sudo chown -R ubuntu /var/www/html
sudo chmod -R 777 /var/www/html
sudo chown -R ubuntu /etc/apache2/sites-available
sudo chmod -R 777 /etc/apache2/sites-available
sudo apt-get install software-properties-common -y
sudo add-apt-repository ppa:ondrej/php
sudo apt-get update -y

sudo apt-get install libpcre3 libpcre3-dev
sudo apt-get install php8.0 php8.0-fpm php8.0-mysql libapache2-mod-php8.0 libapache2-mod-fcgid -y
sudo apt install php8.0-common php8.0-mysql php8.0-xml php8.0-curl php8.0-gd php8.0-imagick php8.0-cli php8.0-dev php8.0-imap php8.0-mbstring php8.0-opcache php8.0-soap php8.0-zip -y

sudo apt install php-pear
sudo pecl channel-update pecl.php.net

sudo pecl install phalcon-5.0.0alpha6

git clone https://github.com/jbboehr/php-psr.git
cd php-psr
phpize
./configure
make
make test

sudo make install
sudo chown -R ubuntu /etc/php/8.0/mods-available/
sudo chmod -R 777 /etc/php/8.0/mods-available/

Created /etc/php/7.4/mods-available/psr.ini with the following lines:
URL : https://forum.phalcon.io/discussion/20343/help-with-installing-phppsr-for-php-74-ubuntu-1804
; configuration for psr module
; priority=20
extension=psr.so

sudo vim /etc/php/8.0/apache2/php.ini
extension=psr.so
extension=phalcon.so 
sudo service apache2 restart

**Comment by niden** [comment]
@RajuHarry Thank you for your effort.

I am going to convert this to a discussion. Installing apache/nginx etc. is not in the scope of Phalcon and as such the installation documentation does not offer any information for that. It is up to the developer/devops individual to have the system set up for Phalcon to be installed.

I have added a small clarification in the docs regarding this.

Thank you again.
